### ISSUE TITLE
Nuke Templates

### ISSUE THREAD

**Issue by lamepennies** [juno-fx/report#191]
Hello,
The templates being used right now are wrong, I thought I uploaded them last August but I guess they were wiped. I'm not sure how to upload now but here they are....
[nuke_templates_fonts.zip](https://github.com/juno-fx/report/files/14169570/nuke_templates_fonts.zip)

**Comment by aldmbmtl** [comment]
@skeleturtle you will need to add this to the pipe repo I think

**Comment by skeleturtle** [comment]
@aldmbmtl  Yup. Im working with Tony on the slate template and have things already getting worked on.

With that said there is a ticket out about a make deploy error for our pipe repo. I'm not sure if that ever got sorted

**Comment by aldmbmtl** [comment]
That ticket is about opening up that location for full write access. That is disabled because this was a live show on CON. I can open it up since it is done.

**Comment by aldmbmtl** [comment]
Permissions opened up on `/pipe`

**Comment by skeleturtle** [comment]
https://github.com/juno-fx/report/issues/175

The only way I can write to the pipe location is to use make deploy. There will be times I need to do this during a show.

**Comment by aldmbmtl** [comment]
That will not be allowed in this case. On dev shows it will be open, not on live shows. You can push it to the /pipe repo which will do the deployment for you. We have this conversation every few months. `/pipe` will always be read-only on live shows.

**Comment by skeleturtle** [comment]
Yes we have this conversation every few months I know.

With this in regards to pipe please make sure this is set up as well. Since I am shifting over logic for show specific scripts/templates to be in an area the leads/sups can set up the show specific nuke template: 
https://github.com/juno-fx/report/issues/181

**Comment by aldmbmtl** [comment]
Based on that other ticket, it seems like there is no need for pipe then?

**Comment by lamepennies** [comment]
Ok, so I had to fix this locally Dan. Inside the gizmo, you'll see an area in read for you to add code for the burnin.

Let me know if this still doesn't work. 
[JunoSlate_v4_local.nk.zip](https://github.com/juno-fx/report/files/14227026/JunoSlate_v4_local.nk.zip)

**Comment by skeleturtle** [comment]
> Ok, so I had to fix this locally Dan. Inside the gizmo, you'll see an area in read for you to add code for the burnin. You'll need to edit the font path in the gizmo as well.
> 
> Let me know if this still doesn't work. [JunoSlate_v4_local.nk.zip](https://github.com/juno-fx/report/files/14227026/JunoSlate_v4_local.nk.zip)

@lamepennies I opened up the file and started getting things back in sync, but I am currently getting hit with connection issues and losing my work. As far as the burn in code, I assume you are just looking for me to add the shot code to the  message knob. Which I'll simply connect that to our parent nodes shot knob. I'll also look to connect things like our thumbnail frame to take the middle frame between what we have set for our inframe/outframe knobs. Overall I'll look to hopefully have things set up this weekend if the connection problems are able to be resolved.

**Comment by skeleturtle** [comment]
@lamepennies 
![image](https://github.com/juno-fx/report/assets/76455095/3ca38453-05ab-4ef9-b90d-f28131cf1677)
I have integrated the slate into our output template and a stand alone template. With that said I have run into a few errors.
Please see attached.


- comments_end/Merg8/7/6/5 are always coming up as an error.
- the Grain2_1.Copy1 non-esistent channel used for mask only comes up when I turn off the burn ins. 

Since I've made multiple updates, that I find myself having done a few times now. If you are able to just point me in the direction as to what these values should be I can make the updates on my end. 

Once thats in place I will begin implementing the slate for our publishes for both our QT and frames

**Comment by lamepennies** [comment]
I believe I fixed this: /shows/CONT/_FromTony

Let me know if this fixes it

**Comment by skeleturtle** [comment]
@lamepennies 

I got the updates, and implemented them into my branches. I started working our slate output generation for publish QT's but currently held up by some of our jfx-luna updates/rework so this will be on hold until that is complete.

**Comment by skeleturtle** [comment]
Things are back up and running from a pipeline side, but leaving this marked as on hold as priority is shifting once again. With that said.

Whats still needed:
- Slate job creation for exrs during publish
- Partial slate logic is set up for Qt Creation
- Anamorphic shows currently will need a custom slate reformatting set up, along with our output  templates reformat
